Mobl_ios_Callisto-master/MealTrackingPilotTests/LocationTests.swift

```diff
@@ -154,7 +154,7 @@ class LocationTests: XCTestCase {
         }
 
         // Server error should result in location being saved to the realm
-        XCTAssertNotNil(location.realm)
+        XCTAssertNotNil(location)
```

Comment:
why we are not checking the location.realm instead of location.

Reply:
Done